common/buildcraft/core/tile/TileEngineRedstone_BC8.java

```diff
@@ -51,12 +51,12 @@ public long minPowerReceived() {
 
     @Override
     public long maxPowerReceived() {
-        return MjAPI.MJ;
+        return 4 * MjAPI.MJ;
```

Comment:
as far as i can see this method isn't even used at all, what's it ment for engine chaining?

Reply:
I must have touched it while trying to fix some engine bug... I'll remove it soonish?